src/ua/javarush/application/CommandLineInterface.java

```diff
@@ -0,0 +1,84 @@
+package ua.javarush.application;
+
+import ua.javarush.constans.Constans;
+import ua.javarush.constans.EncryptionCommandTypes;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Scanner;
+
+public class CommandLineInterface {
+    private final Scanner scanner = new Scanner(System.in);
+    private int key;
```

Comment:
private int key;
    private String pathToFile;
    private int encryptOption;

Can be a local variables, not need to use them as fields